ci/ci/build.py

```diff
@@ -311,9 +311,13 @@ def build(self, batch, code, scope):
 /bin/sh /home/user/convert-cloud-credentials-to-docker-auth-config
 set -x
 
+retry() {{
+    "$@" || (sleep 2 && "$@") || (sleep 4 && "$@")
+}}
+
```

Comment:
Totally down for this, but let's use our `RETRY_FUNCTION_SCRIPT` it is in `hailtop.utils.utils`.

Reply:
I'm all for reusing that, but the retry script requires bash, which the buildkit image doesn't have. I don't think `/bin/sh` is what we want in the retry script because our images have that as `dash`. What do you think is the best way to deal with that?